### PR TITLE
Worked on the withdraw page mobile responsive

### LINKED ISSUE #295: [Frontend] Vault withdraw page [mobile]
🚨 **Attention: Please read the guidelines before applying; otherwise, you will be ignored.** 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy 
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Please make our current **Vault withdraw page** responsive for mobile/tablet devices.
Screenshot 2024-11-28 at 8 51 27

3. Make sure your changes don't break anything in already created desktop version.
4. Verify mobile styles correspond to figma and look exactly the same.
5. Don't implement bottom navigation bar, just a page main content, nothing else.
6. Text me in DM in telegram(djeck_vorobey1) to get access to figma, in your message write your email please

### PR DESCRIPTION
**Pull request by nora360**
Closes #295

### CODE REVIEW

**Review by whateverfw — CHANGES_REQUESTED** [review]
image
Please check screen from 768px and below, currently it is broken and don't correspond to Figma. On some screens like above - everything goes outside a sceeen.

Card styles/padding don't correspond to Figma. Also you can clearly see that 2 cards should fit 1 row on a mobile, but currently it breaks into two lines

Please, go through figma and fix everything accordingly

**Comment by whateverfw** [comment]
top cards don't correspond to figma's sizes
cards in the center have different border radius in figma
image
@nora360

**Review by whateverfw — CHANGES_REQUESTED** [review]
⏫  @nora360

**Comment by nora360** [comment]
@whateverfw  ok working on it

File: frontend/src/components/MetricCard/metricCard.css [hunk 1]
```diff
@@ -9,6 +9,17 @@
   align-items: center;
 }
 
+@media (max-width: 1024px) {
+
+  /* For screens smaller than 1024px */
```
Comment (whateverfw) [inline frontend/src/components/MetricCard/metricCard.css:14]: remove comments

File: frontend/src/components/MetricCard/metricCard.css [hunk 2]
```diff
@@ -9,6 +9,17 @@
...
+  .metric-card {
+    gap: 4px;
+    width: 100%;
+    height: 100px;
+    border-radius: 16px !important;
```
Comment (whateverfw) [inline frontend/src/components/MetricCard/metricCard.css:19]: avoid important

**Comment by whateverfw** [comment]
@nora360 any updates?

**Review by whateverfw — CHANGES_REQUESTED** [review]
@nora360 
image

desktop page is broken now as well, can you please make fixes that works both on mobile and desktop, so desktop looks like in Figma as well?

**Review by whateverfw — CHANGES_REQUESTED** [review]
image
I don't know what is this, but it should be removed, appear on mobile screens at the bottom of a page

**Review by whateverfw — CHANGES_REQUESTED** [review]
I can clearly see a lot of mismatches with figma on mobile screens
That is Figma:
image
And that's broken app:
image

Huge problems and mismatches with padding everywhere, please fix

**Comment by djeck1432** [comment]
@nora360 unassigned you, bad communication ,15 hours ago commited not finished work

**Pull request closed without merging**